ydb/docs/en/core/troubleshooting/examples/schemas/overloaded-shard-1.md

```diff
@@ -0,0 +1,306 @@
+
+#|
+|| Bucket name
+| Latencies, ms
+|
+Transactions,
+single overloaded data shard
+|
+Transactions,
+multiple data shards
+||
+|| 1
+| 0-1
+| 2110
+| ⇧ 16961
+||
+|| 2
+| 1-2
+| 5472
+| ⇧ 13147
+||
+|| 4
+| 2-4
+| 16437
+| ⇩ 6041
+||
+|| 8
+| 4-8
+| 9430
+| ⇩ 432
+||
+|| 16
+| 8-16
+| 98.8
+| ⇩ 52.4
+||
+|| 32
+| 16-32
+| —
+| ⇧ 0.578
+||
+|#
+
+## Testbed {#testbed}
+
+### Topology
+
+For the example, we used a {{ ydb-short-name }} cluster consisting of three servers running Ubuntu 22.04 LTS.
+
+```mermaid
+flowchart
+
+subgraph client[Client VM]
+    cli(YDB CLI)
+end
+
+client-->cluster
+
+subgraph cluster["YDB Cluster"]
+    direction TB
+    subgraph VM1["VM 1"]
+        node1(YDB database node 1)
+        node2(YDB database node 2)
+        node3(YDB database node 3)
+        node4(YDB storage node 1)
+    end
+    subgraph VM2["VM 2"]
+        node5(YDB database node 1)
+        node6(YDB database node 2)
+        node7(YDB database node 3)
+        node8(YDB storage node 1)
+    end
+    subgraph VM3["VM 3"]
+        node9(YDB database node 1)
+        node10(YDB database node 2)
+        node11(YDB database node 3)
+        node12(YDB storage node 1)
+    end
+end
+
+classDef storage-node fill:#D0FEFE
+classDef database-node fill:#98FB98
+class node4,node8,node12 storage-node
+class node1,node2,node3,node5,node6,node7,node9,node10,node11 database-node
+```
+
+### Hardware configuration
+
+Each virtual machine has the following computing resources:
+
+- Platform: Intel Broadwell
+- Guaranteed vCPU performance: 100%
+- vCPU: 28
+- RAM: 32 GB
+
+### Test
+
+The load on the {{ ydb-short-name }} was generated with the `ydb workload` CLI command. For more information, see [{#T}](../../../reference/ydb-cli/commands/workload/index.md).
+
+To reproduce the load, follow these steps:
+
+1. Initialize the tables for the workload test:
+
+    ```shell
+    ydb workload kv init --min-partitions 1 --auto-partition 0
+    ```
+
+    We deliberately disable automatic partitioning for the created tables by using the `--min-partitions 1 --auto-partition 0` options.
+
+1. Emulate the standard workload on the {{ ydb-short-name }} cluster:
+
+    ```shell
+    ydb workload kv run select -s 600 -t 100
+    ```
+
+    We ran a simple load type using a {{ ydb-short-name }} database as a Key-Value storage. Specifically, we used the `select` load to create SELECT queries and get rows based on an exact match of the primary key.
+
+    The `-t 100` parameter is used to ran the test in 100 threads.
+
+
+1. Overload the {{ ydb-short-name }} cluster:
+
+    ```shell
+    ydb workload kv run select -s 1200 -t 250
+    ```
+
+    To simulate the overload, as soon as the first test ended, we ran the same load test in 250 threads.
+
+## See also
+
+- [Troubleshooting performance issues](../../performance/index.md) > [Overloaded shards](../../performance/schemas/overloaded-shards.md)
```

Comment:
I'd make them separate items in the list + throw in some more links, for example, https://ydb-platform--ydb.viewer.diplodoc.com/en/concepts/datamodel/table?revision=pr-14085-00499a683b3d138dd6563f80f9b0d5f0b344d2c4#row-oriented-tables